app/assets/stylesheets/spotlight/_browse.scss

```diff
@@ -38,6 +38,14 @@ $image-overlay-max-height: 300px;
   }
 }
 
+#main-container .browse-group-navigation.nav {
+  margin-bottom: 1rem;
```

Comment:
Can we use `$spacer` here? (also below? `($spacer * .75)` or similar?)

Reply:
Sure (got out of the habit of using that, I guess).